transport_nantes/mobilito/templates/mobilito/mobilito_session_summary.html

```diff
@@ -48,133 +48,131 @@
         {% if mobilito_session.latitude and mobilito_session.longitude %}
             <div id="map" class="col-10 col-md-8 col-lg-6 mx-auto"></div>
         {% endif %}
 
-	    <p>
-		<img src="{% url 'mobilito:mobilito_session_timeseries_image' mobilito_session.session_sha1 %}" alt="{{ mobilito_session.location }}" width="90%">
-	    </p>
-	</div>
-    </div>
-    <div class="row">
-	<div class="col-xs-3">
-	    <p>
-		<img src="{% url 'mobilito:mobilito_session_fraction_image' mobilito_session.session_sha1 %}" alt="{{ mobilito_session.location }}" width="90%">
-	    </p>
+        {# Timeseries #}
+        <div style="overflow: auto;">
+            <img src="{% url 'mobilito:mobilito_session_timeseries_image' mobilito_session.session_sha1 %}"
+            alt="{{ mobilito_session.location }}"
+            width="90%"
+            style="min-width:600px;"
+            class="d-flex mx-auto"
+            >
+        </div>
+    
+        {# Pie chart with labels #}
+        <div class="d-flex flex-wrap col-10 mx-auto">
+            <img src="{% url 'mobilito:mobilito_session_fraction_image' mobilito_session.session_sha1 %}"
+            alt="{{ mobilito_session.location }}"
+            width="90%"
+            class="col-10 col-md-8 mx-auto"
+            style="max-width:400px;"
+            >
+            <ul class="d-flex flex-column col-10 col-md-4 my-auto">
+                <li> {{ mobilito_session.pedestrian_count|default:"0" }} piétons</li>
+                <li> {{ mobilito_session.bicycle_count|default:"0" }} vélos</li>
+                <li> {{ mobilito_session.motor_vehicle_count|default:"0" }} voitures et camions</li>
+                <li> {{ mobilito_session.public_transport_count|default:"0" }}
+                transports collectifs</li>
+            </ul>
         </div>
-	<div class="col-xs-9">
-	    <p>
-		<ul>
-		    <li> {{ mobilito_session.pedestrian_count|default:"0" }} piétons</li>
-		    <li> {{ mobilito_session.bicycle_count|default:"0" }} vélos</li>
-		    <li> {{ mobilito_session.motor_vehicle_count|default:"0" }} voitures et camions</li>
-		    <li> {{ mobilito_session.public_transport_count|default:"0" }}
-			transports collectifs</li>
-		</ul>
-	    </p>
-	</div>
     </div>
-</div>
 
-        {% with start=mobilito_session.start_timestamp end=mobilito_session.end_timestamp %}
-                <div class="dropdown show">
-                    <a href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        <i class="fa-solid fa-ellipsis ml-md-3"></i>
-                    </a>
-                    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink">
-                        <a class="dropdown-item" href="#"
-                        {% if user_has_reported_this_session or request.user == mobilito_session.user.user %}
-                            style="pointer-events: none; color: grey;"
```

Comment:
Would this be better as mobilitain-grey?

Reply:
This is just the font color when one has already flagged a session or is unable to. I don't believe it warrants a special color other than the internet wide grey for disabled stuff :slightly_smiling_face: